### ISSUE TITLE
"Go to collection" button breaks

### ISSUE THREAD

**Issue by horak** [Japan-Digital-Archives/Japan-Digital-Archive#1008]
I thought this problem was reported already but couldn't find it. This is the one that Sebastian and his japanese students were frustrated by. I'm not able to replicate it but it would be great if somebody on our dev team can. So @Japan-Digital-Archives/programmers, it would be awesome if all of you could do try the following:
1. sign in
2. click `collections`
3. click on a random collection
4. drag an item into a collection
5. then click `go to collection`
6. are you running into problems here? If so please report. 

Here are a set of screenshots from Sebastian's laptop:
![unnamed 1](https://cloud.githubusercontent.com/assets/1111424/10420122/c144aedc-7058-11e5-8677-e02b6ae9b010.png)
![unnamed 2](https://cloud.githubusercontent.com/assets/1111424/10420123/c14638c4-7058-11e5-84fc-ed5dc198dffe.png)
![unnamed 3](https://cloud.githubusercontent.com/assets/1111424/10420124/c146c140-7058-11e5-8992-9a468b887df5.png)
![unnamed](https://cloud.githubusercontent.com/assets/1111424/10420125/c1472798-7058-11e5-8e17-37e87f85cf14.png)

**Comment by jdaKoko** [comment]
I tested this problem.  I did not have any problem.

**Comment by jcuanm** [comment]
It is not letting me drag an item into the collection at all

**Comment by horak** [comment]
are you sure you're on the right page @jcuanm, you need to be listing collection items, not collections themselves

**Comment by rchen152** [comment]
Note: Ryo and I were able to reproduce this problem in the tech meeting today by
1) Signing into Koko's account
2) Going to Explore
3) Creating a new collection
4) Dragging items into the collection
5) Clicking Go to collection
(At this point a 404 is triggered)

**Comment by horak** [comment]
@rchen152 great! Did this only happen with @jdaKoko's account? She has many, which one?

**Comment by rchen152** [comment]
@horak Yes, the "Go to collection" button worked normally on my account.
@meeshmon Do you remember what the account name was?

**Comment by meeshmon** [comment]
@horak @rchen152 koko howell account

**Comment by horak** [comment]
what's the username for that exactly?

**Comment by meeshmon** [comment]
@horak @rchen152 "koko howell" account

**Comment by horak** [comment]
@rchen152 do you think you could try investigating if this happens with any of her other accounts and what may be the difference between her accounts?

**Comment by meeshmon** [comment]
@jdaKoko Can you try with your other accounts to see if there is a 404 error with your other account 
(try to make a new collection, throw a few items in and try going to the collection)?

**Comment by jdaKoko** [comment]
@horak @rchen152 @meeshmon 
I cannot sign into all of my accounts.  I cleared browsing data and restarted my computer.

**Comment by jdaKoko** [comment]
@meeshmon @rchen152 @horak 
I was able to sing in Showa Boston account, but not Kim, Koko Howell, JDA,KH accounts.
I do not have the 404 error problem, using Showa Boston account.

**Comment by jdaKoko** [comment]
When I changed my Wifi Network to connect the internet, I was able to sign in all of my JDA accounts.
I created new collections in each account, dragged items and opened the newly created collections in each account. 
I do not have the 404 error problems in any of them.

**Comment by horak** [comment]
Hmmm, so from what I'm getting, this problem is just very inconsistent, that of course makes it a lot more difficult to diagnose. Are there any problems that you have consistently?

**Comment by jdaKoko** [comment]
The problems I have are already posted in the GitHub.
I have not heard about this issue.

**Comment by horak** [comment]
This problem though you can't consistently replicate though? It comes and goes?

**Comment by jdaKoko** [comment]
I never experienced this problem.  I do not know who's issue this was.

**Comment by horak** [comment]
Unfortunately apparently a lot of Tohoku university students and Sebastian had this happen. I really want to get to bottom of it since it seemed to have been a major reason for them having usability issues with the archive this past semester.

**Comment by horak** [comment]
Can all the @Japan-Digital-Archives/programmers should try this to see if they can replicate?
